arch/arm/boot/dts/overlays/revpi-connect4-overlay.dts

```diff
@@ -273,6 +273,7 @@
 			pinctrl-0 = <&rs485_pins>;
 			status = "okay";
 			linux,rs485-enabled-at-boot-time;
+      rs485-rts-active-low;
```

Comment:
Wrong indention type (pleasse use tabs)

Reply:
Also there should be at least one sentence of commit message